src/pages/guardian-directory.tsx

```diff
@@ -16,12 +19,42 @@ const GuardianDirectory = () => {
   const [directory, setDirectory] = useState(Array<GuardianDirectoryType>)
   const { data } = useGetGuardianDirectoryQuery()
 
+  const [searchQuery, setSearchQuery] = useState('')
+  const { data: searchData, loading: loadingSearch } =
+    useSearchGuardianDirectoryQuery({
+      variables: { search: searchQuery },
+    })
+
+  useEffect(() => {
```

Comment:
Please feel free to put forth any other approaches for how to handle updating the search results on the front-end. I know we didn't have time to review at eng sync before my vacation.

What all of this is doing:

* When the search form is submitted, it sets the `searchQuery` state.
* When the `searchQuery` state changes, that causes the `useSearchGuardianDirectoryQuery` hook to re-run and grab the search results
* When the searchQuery or searchData update, a `useEffect` sets the directory state to those results, re-rendering the table to only show the results.